README.md

```diff
@@ -6,17 +6,17 @@ React lightweight library for anchor scrolling and navigation tied to URL hash.
 
 The library export multiples helpers designed to make your anchor navigation works seamlessly.
 
-Two hooks :
+And four components :
 
-- useAnchorScrollListener
-- useHash
+- [AnchorContext](####AnchorContext)
+- [AnchorLink](####AnchorLink)
+- [AnchorProvider](####AnchorProvider)
+- [AnchorSection](####AnchorSection)
```

Comment:
It seems like all links in the page are incorrects, the target only require one hash.

Reply:
Indeed, vscode markdown preview handled this case 👍

Reply:
4e8607e